tests/unit/libbpf_test.cpp

```diff
@@ -1204,3 +1204,148 @@ TEST_CASE("bpf_object__load", "[libbpf]")
     REQUIRE(bpf_link__destroy(link) == 0);
     bpf_object__close(object);
 }
+
+// Test bpf() with the following command ids:
+// BPF_PROG_LOAD, BPF_OBJ_GET_INFO_BY_FD, BPF_PROG_GET_NEXT_ID,
+// BPF_MAP_CREATE, BPF_MAP_GET_NEXT_ID, BPF_PROG_BIND_MAP,
+// and BPF_MAP_GET_FD_BY_ID.
+TEST_CASE("BPF_PROG_BIND_MAP", "[libbpf]")
```

Comment:
Minor: should we rename the test case name, as it looks like right now that it is only testing `BPF_PROG_BIND_MAP`, but it is actually testing many other IDs.